03/Task03.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
+import github from '../src/modules/github/github';
+import Github from '../src/modules/github/github';
```

Comment:
Uwaga na duplikaty. Może to Cię zainteresuje: https://devmentor.pl/b/git-nie-ogarnia-jak-zmienic-wielkosc-liter-w-nazwach-plikow-i-katalogow